vhd/packer/install-dependencies.sh

```diff
@@ -424,6 +424,7 @@ echo "  - busybox" >> ${VHD_LOGS_FILEPATH}
 K8S_VERSIONS="
 1.17.2
 1.17.1
+1.16.6
```

Comment:
Let's remove the 1.16.1 versions from the VHD spec

Reply:
I removed 1.16.2 as well, since it's been disabled forever.